src/Chapter15/Listing15.17.OrderingWithSystem.Linq.Enumerable.OrderByAndThenBy.cs

```diff
@@ -23,6 +25,9 @@ public static void Main()
                 .ThenByDescending(
                     patent => patent.Title);
             Print(items);
+
+            //...
+            #endregion INCLUDE
```

Comment:
Is this in the right place? I would have expected it after the closing brace on line 40